contents/docs/cdp/batch-exports/postgres.md

```diff
@@ -16,6 +16,8 @@ Batch exports can be used to export data to a Postgres table.
 
 > **Note:** Wherever your Postgres database is hosted, make sure the host is set to accept all incoming connections so that PostHog can connect to the database and insert events. PostHog does not guarantee a static list of IP addresses to whitelist. If this is not possible in your case, consider exporting data to a different destination (like [S3](./s3.md)) and then setting up your own system for getting data into your Postgres database.
 
+> **Note:** We only support connections using SSL/TLS.
```

Comment:
Fixing a link and using a list of notes instead of two separate ones
```suggestion
> **Notes:** 
> - Wherever your Postgres database is hosted, make sure the host is set to accept all incoming connections so that PostHog can connect to the database and insert events. PostHog does not guarantee a static list of IP addresses to whitelist. If this is not possible in your case, consider exporting data to a different destination (like [S3](/docs/cdp/batch-exports/s3)) and then setting up your own system for getting data into your Postgres database.
>
> -  We only support connections using SSL/TLS.
```